test/interface.jl

```diff
@@ -82,6 +82,11 @@ using Zygote: Grads
     @test ps isa Params
     @test issetequal(ps, Set([y]))
   end
+
+  @testset "constructor with empty args" begin
+    @test length(Params()) == 0
+    @test length(Params(())) == 0
```

Comment:
```suggestion
    @test length(Params([])) == 0
```
Params typically expects a mutable container.

Reply:
we do suuport tuples though https://github.com/FluxML/Zygote.jl/blob/4d22edf4a27db0302badb12e1faec6d5ee31fd9d/src/compiler/interface.jl#L142

Reply:
Nope, notice that `collect` will convert it to an array.

Reply:
so? We support constructing Params out of tuples e.g. `Param((w, b))